components/automate-gateway/handler/tokens.go

```diff
@@ -1,93 +0,0 @@
-package handler
-
-import (
```

Comment:
where does the v2 tokens handler live?

Reply:
💁 https://github.com/chef/automate/blob/master/components/automate-gateway/handler/iam/v2/tokens/tokens.go